psamm_import/importer.py

```diff
@@ -168,19 +171,98 @@ def model_compounds(model):
         yield d
 
 
-def model_reactions(model, exchange=False):
-    """Yield model reactions as YAML dicts"""
+def reactions_to_files(model, dest, yaml_args, exchange,
+                       single_reaction_file):
+    """Turns the reaction subsystems into their own files.
 
-    for reaction_id, reaction in sorted(iteritems(model.reactions)):
+    If a subsystem has a number of reactions over the threshold, it gets its
+    own yaml file. All other reactions, those that don't have a subsystem or
+    are in a subsystem that falls below the threshold, get added to a common
+    reaction file.
+
+    Args:
+        model: :class:psamm_import.model.MetabolicModel.
```

Comment:
I think you are missing backticks here. It should be ``:class:`psamm_import...` ``.